nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Aj active

# Description
Enabled the ability to promote and demote users along with the ability for an admin to activate and deactivate 
Fixes #25 #26 #27 
## Type of change
# Testing Instructions
- Pull down PR branch
- Make sure sql express and appsettings local file are updated
- Launch API and App
- Navigate to Profiles
-At profiles, click "Go to inactive", this should take you to the inactive profiles.
-Click, Activate, this will take you to a page where you can confirm
- on cancel you will be taken back to the list of inactive users and the user selected will no longer appear.
-CLick another Activate list item, once on the page click, Cancel. This should take you back to inactive profiles.
-click the edit button, here you can change the users type.
-click promote or demote, it will only let you click once.
- after action you will be taken back to the list of user profiles.
-click another edit button, once on the edit back click back.
# Checklist:
